transmogrifier/sources/marc.py

```diff
@@ -225,38 +230,43 @@ def get_optional_fields(self, xml: Tag) -> dict:
             holding_call_number_value = (
                 self.create_subfield_value_string_from_datafield(datafield, ["bb"])
             )
-            holding_collection_value = ", ".join(
-                [
-                    holdings_collection_crosswalk.get(
-                        holding_collection_value, holding_collection_value
-                    )
-                    for holding_collection_value in (
-                        self.create_subfield_value_list_from_datafield(datafield, "i")
-                    )
-                ]
-            )
-            holding_format_value = ", ".join(
-                [
-                    holdings_format_crosswalk.get(
-                        holding_format_value, holding_format_value
-                    )
-                    for holding_format_value in (
-                        self.create_subfield_value_list_from_datafield(datafield, "t")
-                    )
-                ]
-            )
-            holding_location_value = ", ".join(
-                [
-                    holdings_location_crosswalk.get(
-                        holding_location_value, holding_location_value
-                    )
-                    for holding_location_value in (
-                        self.create_subfield_value_list_from_datafield(
-                            datafield, ["aa"]
-                        )
-                    )
-                ]
-            )
+            crosswalked_collection_values = []
```

Comment:
This block is very repetitive but I thought it was more readable, let me know if you see a more efficient approach